deployment/clouddeploy/gke-workers/environments/oss-vdb/make-bugs-public.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: make-bugs-public
+spec:
+  schedule: "0 12 * * *"
+  concurrencyPolicy: Forbid
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          containers:
+          - name: cron
+            image: cron
+            imagePullPolicy: Always
+            command: ["/usr/local/bin/make_bugs_public/make_bugs_public.py"]
+            resources:
```

Comment:
You should define the `GOOGLE_CLOUD_PROJECT` env here as well - I think ndb uses it